src/main/groovy/val/Checkers.groovy

```diff
@@ -16,6 +16,10 @@ import java.util.regex.Pattern
  */
 class Checkers {
 
+    //Hack to provide completion
+    @Delegate()
+    Idator dummyMethods = new Idator()
```

Comment:
I'm planning on defining the Idator methods in an interface and then have this object be an implementation that is an angry sentinel and just throws...something, probably `UnsupportedOperationException`